lib/extras/packed_image.h

```diff
@@ -64,6 +64,12 @@ class PackedImage {
   // Whether the y coordinate is flipped (y=0 is the last row).
   bool flipped_y = false;
 
+  // Whether the range is determined by format or by JxlBasicInfo
+  // e.g. if format is UINT16 and JxlBasicInfo bits_per_sample is 10,
+  // then if bitdepth_from_format == true, the range is 0..65535
+  // while if bitdepth_from_format == true, the range is 0..1023.
```

Comment:
Shouldn't that be:
`// while if bitdepth_from_format == false, the range is 0..1023.`?

Reply:
Ah yes indeed. Good catch!